docs/domains/DataAccess/services/DataAcquisitionService/openapi.yml

```
@@ -973,6 +973,186 @@ paths:
+          type: string
+      responses:
+        "204":
+          description: No Content
+        "404":
+          description: Facility Not Found
+        "500":
+          description: Internal Server Error
+  /location-mapping:
+    get:
+      tags:
+      - location-mapping
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)